app/src/shared/components/Sidebar/index.jsx

```diff
@@ -64,8 +64,25 @@ const Sidebar = ({ className, isOpen, onClose, children }: Props) => {
 
 export { Select }
 
+const Container = styled.div`
+    padding: 24px 32px;
```

Comment:
`Sidebar.Container` (in this file "just" `Container`) defines spacing around all key sidebar components. It's used all over the place. Reasonable DRY FTW!